### PR TITLE
common-arguments, git: add Malayalam translation

### PR DESCRIPTION
**Pull request by akash1729**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by blueskyson — APPROVED** [review]

File: pages.ml/common/git.md
Comment (kbdharun) [inline pages.ml/common/git.md:5]:
```suggestion
```

File: pages.ml/common/git.md
Comment (kbdharun) [inline pages.ml/common/git.md:4]:
```suggestion
> `commit`, `add`, `branch`, `checkout`, `push` പോലുള്ള ഉപകമാന്റുകൾക്ക് പ്രത്യേക ഡോക്യൂമെന്റഷൻ ഉണ്ട്. അതിനായി `tldr git {{ഉപകമാൻഡ്}}` ഉപയോഗിക്കുക.
```

File: pages.ml/common/git.md
Comment (kbdharun) [inline pages.ml/common/git.md:30]: Can you translate the `value` command argument too?

**Review by kbdharun — COMMENTED** [review]
Welcome to tldr, Thanks for your contribution. I have some suggestions for this page.

**Comment by kbdharun** [comment]
If possible can you add Malayalam words to [common-arguments](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md) as it will be a helpful reference for reviewers and other translators?

File: pages.ml/common/git.md
Comment (kbdharun) [reply pages.ml/common/git.md:29]: Would this be better @akash1729 ?

```suggestion
`git -c '{{config.key}}={{മൂല്യം}}' {{ഉപകമാൻഡ്}}`
```

**Comment by kbdharun** [comment]
@sanbabyfrancis can you review this PR?

> Thanks in advance.

**Comment by kbdharun** [comment]
@navarroaxel, How to resolve the conflict in the branch?

**Comment by navarroaxel** [comment]
> @navarroaxel, How to resolve the conflict in the branch?

The author should merge `main` into this PR

File: pages.ml/common/git.md
Comment (akash1729) [reply pages.ml/common/git.md]: yeah മൂല്യം would be better

**Review by sanbabyfrancis — COMMENTED** [review]
`common-arguments.md` looks good! @kbdharun 

Just a suggestion @akash1729 , in `git.md` can you modify the verb in line 11 and line 27 with `കാണുവാൻ` and `ചെയ്യുവാൻ` respectively? It would ensure a common linguistic tone throughout the cheatsheet. Thanks.

**Comment by akash1729** [comment]
> `common-arguments.md` looks good! @kbdharun
> 
> Just a suggestion @akash1729 , in `git.md` can you modify the verb in line 11 and line 27 with `കാണുവാൻ` and `ചെയ്യുവാൻ` respectively? It would ensure a common linguistic tone throughout the cheatsheet. Thanks.

"Yeah that makes sense. I have changed it"

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: contributing-guides/translation-templates/common-arguments.md
Comment (kbdharun) [inline contributing-guides/translation-templates/common-arguments.md:24]:
```suggestion
| ml    |പാത്ത്/ടു/ഫയൽ              |പാത/ടു/ഡയറക്‌ടറി             | പാത്ത്/ടു/ഫയൽ_ഓർ_ഡയറക്‌ടറി           |പാക്കേജ്      |ഉപയോക്തൃനാമം         |
```
@sanbabyfrancis @akash1729 Would this be better?
> Note: I translated this using Google Translate to add `to/` to paths, feel free to make changes.

Comment (akash1729) [reply contributing-guides/translation-templates/common-arguments.md:24]: Yeah it would make sense to keep it പാത്ത്/ടു since the other one is kind of ambiguous. I have changed it

Comment (sanbabyfrancis) [reply contributing-guides/translation-templates/common-arguments.md:24]: 1. `പാത/` is a native way of representing `path/to/` in Malayalam. 
2. `പാത്ത്/ടു` is simply the Malayalam translation of the words: `path` and `to`.

Both sound right in our context. If we were to strictly follow the style: `path/to/`, let's go with the second one.

Comment (kbdharun) [reply contributing-guides/translation-templates/common-arguments.md:24]: I am fine with anything. @navarroaxel What do you think?

Comment (sanbabyfrancis) [reply contributing-guides/translation-templates/common-arguments.md:24]: Also, in the above edit, replace `പാത/ടു/ഡയറക്‌ടറി` with `പാത്ത്/ടു/ഡയറക്‌ടറി` @kbdharun

Comment (abrahamparayil) [reply contributing-guides/translation-templates/common-arguments.md:24]: @kbdharun @akash1729 What do you guys think about `ഫയലിലേക്കുള്ള/പാത`, `ഡയറക്ടറിയിലേക്കുള്ള/പാത` and `ഫയലിലേക്കോ_ഡയറക്ടറിയിലേക്കോ/ഉള്ള/പാത`? I think we should translate the strings where possible rather than transliterate.

File: pages.ml/common/git.md
Comment (abrahamparayil) [inline pages.ml/common/git.md:4]:
```suggestion
> പ്രോഗ്രാമുകളുടെ പല പതിപ്പുകൾ പലയിടങ്ങളിലായി സൂക്ഷിക്കുവാനും നിയന്ത്രിക്കുവാനും ഉള്ള വികേന്ദ്രീകൃത പതിപ്പ് നിയന്ത്രണ സംവിധാനം
> `commit`, `add`, `branch`, `checkout`, `push` മുതലായ ചില ഉപകമാൻഡുകൾക്ക് അവരുടേതായ ഡോക്യുമെന്റേഷൻ ഉണ്ട്, `tldr git {{ഉപകമാൻഡ്}}` വഴി അവ കാണാൻ കഴിയും.
```

File: pages.ml/common/git.md
Comment (abrahamparayil) [inline pages.ml/common/git.md:7]:
```suggestion
- നിങ്ങൾ ഉപയോഗിക്കുന്ന ഗിറ്റിന്റെ പതിപ്പ് പരിശോധിക്കാൻ:
```

File: pages.ml/common/git.md
Comment (abrahamparayil) [inline pages.ml/common/git.md:23]:
```suggestion
- ഒരു ഇഷ്‌ടാനുസൃത ശേഖരണത്തിന്റെ/റിപ്പോസിറ്ററിയുടെ റൂട്ട് പാതയിൽ ഒരു Git സബ്‌കമാൻഡ് എക്‌സിക്യൂട്ട് ചെയ്യാൻ:
```

File: pages.ml/common/git.md
Comment (abrahamparayil) [inline pages.ml/common/git.md:25]:
```suggestion
`git -C {{ശേഖരണത്തിലേക്കുള്ള/പാത}} {{ഉപകമാൻഡ്}}`
```

File: pages.ml/common/git.md
Comment (abrahamparayil) [inline pages.ml/common/git.md:27]:
```suggestion
- ഒരു കോൺഫിഗറേഷൻ സെറ്റ് ഉപയോഗിച്ച് Git സബ്കമാൻഡ് എക്സിക്യൂട്ട് ചെയ്യാൻ:
```

**Review by abrahamparayil — CHANGES_REQUESTED** [review]
I've added a few comments.

@akash1729 @sanbabyfrancis @kbdharun Let me know what you guys think.

File: contributing-guides/translation-templates/common-arguments.md
Comment (sanbabyfrancis) [reply contributing-guides/translation-templates/common-arguments.md:24]: This looks perfect! It is precise, grammatically structured and more readable than the transliterated string. Maybe we should adopt this instead of compromising the readability of the text by sticking to the traditional transliterated `path/to/directory` string.

Comment (kbdharun) [reply contributing-guides/translation-templates/common-arguments.md:24]: > @kbdharun @akash1729 What do you guys think about `ഫയലിലേക്കുള്ള/പാത`, `ഡയറക്ടറിയിലേക്കുള്ള/പാത` and `ഫയലിലേക്കോ_ഡയറക്ടറിയിലേക്കോ/ഉള്ള/പാത`? I think we should translate the strings where possible rather than transliterate.

Thanks, I will update my original review.

File: pages.ml/common/git.md
Comment (akash1729) [reply pages.ml/common/git.md:27]: for subcommand should we use ഉപകമാൻഡ് or സബ്കമാൻഡ് ? I have used ഉപകമാൻഡ് everywhere else

File: contributing-guides/translation-templates/common-arguments.md
Comment (akash1729) [reply contributing-guides/translation-templates/common-arguments.md:24]: Yeah this is perfect. This can use this approach.
Should we use `ഫയലിലേക്ക്/ഉള്ള/പാത` or `ഫയലിലേക്കുള്ള/പാത` ?
What do you guys think ?

Comment (kbdharun) [reply contributing-guides/translation-templates/common-arguments.md:24]: > Yeah this is perfect. This can use this approach. Should we use `ഫയലിലേക്ക്/ഉള്ള/പാത` or `ഫയലിലേക്കുള്ള/പാത` ? What do you guys think ?

I think we can go with this ` ഫയലിലേക്കുള്ള/പാത` as discussed above. You can commit the suggestion I made above.

File: pages.ml/common/git.md
Comment (kbdharun) [reply pages.ml/common/git.md:27]: I think we can go with `സബ്കമാൻഡ്` but I can't find the `subcommand` used already on any pages. So, I guess we can choose anything.

**Comment by kbdharun** [comment]
- **To-do:** After this PR is merged we need to update some paths in @sanbabyfrancis old PRs.
- Note to maintainer: I think we can leave Co-Authored by in this commit message as multiple contributors have worked on this PR.

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**
